src/views/client-dashboard/ClientPayments.vue

```diff
@@ -10,7 +10,34 @@
             :class="`${card.primary ? 'active' : ''} client-payment-card-item`"
             @click.prevent="onSelect(card)"
           >
-            <div class="card-number" v-html="getCardInfo(card)" />
+            <div class="d-flex align-items-center">
+              <img
+                v-if="['mastercard', 'MasterCard'].includes(card.brand)"
+                src="@/assets/svg/mastercard.svg"
+                alt="mastercard"
+                width="30"
+                height="30"
+                class="lazy ml-n1 mr-1"
+              >
+              <img
+                v-else-if="['visa', 'Visa'].includes(card.brand)"
+                src="@/assets/svg/visa.svg"
+                alt="visa"
+                width="30"
+                height="30"
+                class="lazy ml-n1 mr-1"
+              >
+              <img
```

Comment:
From what I see, there are 3 type of options, but the structure is identical. So what you could do, is to have a computed property or a method, that is creating the 3 variants for you: 

such as:

`getCardInfo.src`, `getCardInfo.alt`, where in the getCardInfo, you can do:

`getCardInfo() {
if or switch - add the logic that you have in the template
Based on the logic, return the correct src and alt
`


}

Reply:
ok, I did some changes here also. Now we have just a single <img> tag with dynamic :src and :alt

Reply:
great